modules/migration-configuring-mcg.adoc

```diff
@@ -17,19 +17,19 @@ You must retrieve the Multicloud Object Gateway (MCG) credentials in order to cr
 ifdef::installing-oadp-mcg[]
 endif::[]
```

Comment:
Interesting, this...it was in here before this PR but not sure it's doing anything. I see from build job logs that the `installing-oadp-mcg` ID shows up a lot. Not sure that this ifdef is causing those fails in any way. 🤷 
![image](https://user-images.githubusercontent.com/47002677/161170407-8ab514e9-ef1d-41ce-b648-cff9d799a39a.png)

Reply:
I commented out the back to back `ifdef` `endif` and the build worked. interesting since it was there before? if you're okay with this then I am too!

Reply:
Since it's working, I'm okay with it for now but I'd suggest following up with a separate PR to remove the conditionals altogether and see if it builds okay still.